src/docs/resources/faq.md

```diff
@@ -401,22 +495,20 @@ on the edge.
 
 ### Does Flutter run on the web?
 
-Yes, web support is currently available in the [beta][beta channel]. You can
-compile existing Flutter code to work on the web, but at present not all
-platform plug-ins have a web equivalent. For more details, check out the [web
-instructions][].
+Yes, web support is available in the stable channel.
+You can compile existing Flutter code to work on the web.
+For more details, check out the [web instructions][].
 
 ### Can I use Flutter to build desktop apps?
 
-Yes, desktop support is in preview for Windows, macOS, and Linux. The current
-progress is documented on the [Desktop][desktop] page.
+Yes, desktop support is in beta for Windows, macOS, and Linux.
```

Comment:
Do we want to mention that there is a beta snapshot included in stable, with usage gated behind a flag? 

/cc @csells

Reply:
I had mentioned it elsewhere, but I added it here, too.